migration/migrator/data/course_tables.sql

```diff
@@ -417,6 +417,20 @@ CREATE TABLE peer_assign (
     user_id character varying NOT NULL
 );
 
+--
+-- Name: peer_feedback; Type: TABLE; Schema: public; Owner: -
+--
+
+CREATE TABLE peer_feedback (
+    pf_id integer NOT NULL,
+    grader_id character varying(255) NOT NULL,
+    user_id character varying(255),
+    team_id character varying(255),
+    g_id character varying(255) NOT NULL,
+    feedback character varying(255),
+    CONSTRAINT user_team_id_check CHECK (user_id IS NOT NULL OR team_id IS NOT NULL)
+);
```

Comment:
See my previous review for how I think we should restructure this table. (Remove `pf_id` and `team_id`).

Reply:
Removed pf_id and team_id